modules/codegen/src/main/scala/com/twilio/guardrail/generators/ScalaParameter.scala

```diff
@@ -140,7 +140,7 @@ object ScalaParameter {
       resolved <- SwaggerUtil.ResolvedType.resolve[L, F](meta, protocolElems)
       SwaggerUtil.Resolved(paramType, _, baseDefaultValue) = resolved
 
-      required = parameter.getRequired()
+      required = Option[java.lang.Boolean](parameter.getRequired()).fold(false)(identity)
```

Comment:
I still prefer having extension methods defined for swagger model entities to reduce boilerplate for such occasions

Reply:
I'm in favor as well. My goal after this is to push all `OpenAPI` knowledge to its own algebra, so future changes can be done from the outside without exposing so much knowledge (`Option`, exceptions, `$ref`, tree-walking) into the core generators. I didn't know this was absolutely necessary before starting this work, but now I'm leaning very heavily in favor of that strategy. It's too much noise right now.